src/DynamoAnalyzer/Models/DuplicatedPackage.cs

```diff
@@ -0,0 +1,24 @@
+namespace DynamoAnalyzer.Models
+{
+    /// <summary>
+    /// Represents a duplicated DLL and the numbers of times that other packages references a DLL with the same name
+    /// </summary>
+    public class DuplicatedPackage
+    {
+        public string ArtifactName { get; set; }
+        public int Count { get; set; }
+        public string[] Packages { get; set; }
+
+        public DuplicatedPackage()
```

Comment:
why is this required? For JSON? If so please add a comment so no one removes it.

Reply:
After all the packages are analyzed, this dto can show which dll names are common in other packages, so, in the future, we can speed up the analysis by excluding dlls that could be nuget packages and no created by the package author, this dto is used to write values to a csv file